packages/medusa/src/loaders/medusa-app.ts

```diff
@@ -0,0 +1,58 @@
+import { MedusaApp, ModulesDefinition } from "@medusajs/modules-sdk"
+
+import { CommonTypes } from "@medusajs/types"
+import { ContainerRegistrationKeys } from "@medusajs/utils"
+import { asValue } from "awilix"
+import { joinerConfig } from "../joiner-config"
+import { mergeModulesConfig } from "../utils/merge-modules-config"
+import modulesConfig from "../modules-config"
+import { remoteQueryFetchData } from ".."
+
+export const loadMedusaApp = async (
+  {
+    configModule,
+    container,
+  }: { configModule: CommonTypes.ConfigModule; container: any },
+  config = { register: true }
+) => {
+  mergeModulesConfig(configModule.modules ?? {}, modulesConfig)
+
+  const injectedDependencies = {
+  }
```

Comment:
question: we moved from having the pg connection in injected deps to now passing the clientUrl in shared resources – curious to know why?

Also, do we need a todo or something here for the empty object?

Reply:
It should be the connection, otherwise it is not shared anymore